src/com/facebook/buck/jvm/java/AccumulateClassNamesStep.java

```diff
@@ -163,10 +158,9 @@ public InputStream openStream() throws IOException {
     Map<String, HashCode> classNames = new HashMap<>();
 
     for (String line : lines) {
-      List<String> parts = CLASS_NAME_AND_HASH_SPLITTER.splitToList(line);
-      Preconditions.checkState(parts.size() == 2);
-      String key = parts.get(0);
-      HashCode value = HashCode.fromString(parts.get(1));
+      int index = line.trim().lastIndexOf(CLASS_NAME_HASH_CODE_SEPARATOR);
```

Comment:
The trim probably isn't necessary and creates a copy. Just precondition check on index > 0